frontend/Dockerfile

```diff
@@ -3,7 +3,8 @@
       FROM node-s2i as build
       WORKDIR /tmp
       COPY . /tmp
-      RUN yarn && yarn build --openssl-legacy-provider --prod
+      RUN yarn
```

Comment:
should the 'deploy_locally' instructions also say to use `yarn` and `yarn start` for development instead of `npm i && npm run start` ?

Reply:
yes i made a comment about this in your pr that you recently submitted